### PR TITLE
chore(litmus-portal): Code refactoring

### PR DESCRIPTION
**Pull request by arkajyotiMukherjee**
**What this PR does / why we need it**: Code refactoring and Code quality improvements
Fixed: Maps, Login page and minor changes in other components and linting rules

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Checklist**

* [ ] Does this PR have a corresponding GitHub issue?
* [ ] Have you included relevant README for the chaoslib/experiment with details?
* [ ] Have you added debug messages where necessary? 
* [ ] Have you added task comments where necessary? 
* [ ] Have you tested the changes for possible failure conditions?
* [ ] Have you provided the positive & negative test logs for the litmusbook execution?
* [ ] Does the litmusbook ensure idempotency of cluster state?, i.e., is cluster restored to original state?
* [ ] Have you used non-shell/command modules for Kubernetes tasks?
* [ ] Have you (jinja) templatized custom scripts that is run by the litmusbook, if any? 
* [ ] Have you (jinja) templatized Kubernetes deployment manifests used by the litmusbook, if any?
* [ ] Have you reused/created util functions instead of repeating tasks in the litmusbook?
* [ ] Do the artifacts follow the appropriate directory structure? 
* [ ] Have you isolated storage (eg: OpenEBS) specific implementations, checks? 
* [ ] Have you isolated platform (eg: baremetal kubeadm/openshift/aws/gcloud) specific implementations, checks?  
* [ ] Are the ansible facts well defined? Is the scope explicitly set for playbook & included utils?
* [ ] Have you ensured minimum/careful usage of shell utilities (awk, grep, sed, cut, xargs etc.,)?
* [ ] Can the limtusbook be executed both from within & outside a container (configurable paths, no hardcode)?
* [ ] Can you suggest the minimal resource requirements for the litmusbook execution?
* [ ] Does the litmusbook job artifact carry comments/default options/range for the ENV tunables?
* [ ] Has the litmusbooks been linted? 

**Special notes for your reviewer**:

### CODE REVIEW

File: litmus-portal/frontend/src/pages/LoginPage/index.tsx [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-import useStyles from './styles';
+import { Link } from 'react-router-dom';
 import config from '../../config';
 import useActions from '../../redux/actions';
 import * as UserActions from '../../redux/actions/user';
 import { history } from '../../redux/configureStore';
+import useStyles from './styles';
 
 interface authData {
   username: string;
```
[the remark this answers is not in the document]
Comment (arkajyotiMukherjee) [reply litmus-portal/frontend/src/pages/LoginPage/index.tsx:11]: I think GitHub is folding the fold...unfold it and se...the 1st one is an interface and 2nd is the implementation of the interface @rajdas98

**Comment by oumkale** [comment]
./preview

File: litmus-portal/frontend/src/components/BinarySwitch/index.tsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    },
+    track: {
+      border: `1px solid ${theme.palette.grey[500]}`,
+      borderRadius: 16 / 2,
+      opacity: 1,
+      backgroundColor: theme.palette.common.white,
+    },
+    checked: {},
+  })
+)(Switch);
```
Comment (Saranya-jena) [inline litmus-portal/frontend/src/components/BinarySwitch/index.tsx:39]: Can we add a separate 'styles' file for this and proper units to height, width, margin, etc?

Comment (arkajyotiMukherjee) [reply litmus-portal/frontend/src/components/BinarySwitch/index.tsx:39]: Separate styles won't work here as this is a modified one...you can check the mui docs but yes the height and width I'll change it slipped my mind

Comment (Saranya-jena) [reply litmus-portal/frontend/src/components/BinarySwitch/index.tsx:39]: Okay, didn't know that. Thanks!

**Pull request merged**